core/src/types/request.rs

```diff
@@ -105,6 +105,21 @@ mod tests {
 		);
 	}
 
+	#[test]
+	fn call_serizalize_without_params() {
```

Comment:
```suggestion
	fn call_serialize_without_params() {
```

Reply:
Yes, sure, I'll add that test case. Probably, I can do it tomorrow.